postgres/osmap.yaml

```diff
@@ -85,15 +90,30 @@ RedHat:
 
 {% else %}
 
+  {% set data_dir = '/var/lib/pgsql/data' %}
   pkg: postgresql-server
   pkg_client: postgresql
 
 {% endif %}
 
+  service: postgresql-{{ repo.version }}
+  pkgs_extra: [ postgresql-contrib, postgresql-plpython, postgresql-jdbc, postgresql-docs]
+  conf_dir: {{ data_dir }}
+  prepare_cluster:
+    command: initdb --pgdata='{{ data_dir }}'
+    test: test -f '{{ data_dir }}/PG_VERSION'
+    user: postgres
```

Comment:
All these lines are unnecessary, please see `defaults.yaml`.

Reply:
Yes.  But 'initdb' and 'data_dir' can differ between OS in theory.  How can this be addressed?

Reply:
Here you just moved the section that should be defined for upstream PostgreSQL distribution.
If these parameters are not defined, the corresponding ones from `defaults.yaml` would be merged in.